src/helpers/articleHelper.js

```diff
@@ -5,17 +5,17 @@ const { Op } = Sequelize;
 const { Users, Categories, Articles } = models;
 
 /**
-   * filter for search article
```

Comment:
So I saw that we have another file called `article` in this helpers folder. I don't really understand why the methods here were not added to that file (I'd rather they were). Is there a reason why we decided to create another file called `genericArticleHelper`? Also, I need to understand why you feel the need to use the word `generic` in the codebase. I have seen you describe functions as `generic ...` in your JSDocs. Is there a specific reason for this?

Reply:
Before the search article PR was merged the other article file wasn't there. I called it ```genericArticleHelper``` because the sequelize operators used in the file are not limited. it can be reused anywhere to find any article or author.

Reply:
Okay. Please move all methods in this file to the `./helpers/article.js` file. Also, I don't think we need to use generic. Helpers can generally be used anywhere so there's no need to specify. Thanks